Libraries/src/Amazon.Lambda.Serialization.SystemTextJson/DefaultLambdaJsonSerializer.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using Amazon.Lambda.Core;
+using Amazon.Lambda.Serialization.SystemTextJson.Converters;
+
+namespace Amazon.Lambda.Serialization.SystemTextJson
+{
+    /// <summary>
+    /// Custom ILambdaSerializer implementation which uses System.Text.Json
+    /// for serialization.
+    /// 
+    /// <para>
+    /// If the environment variable LAMBDA_NET_SERIALIZER_DEBUG is set to true the JSON coming
+    /// in from Lambda and being sent back to Lambda will be logged.
+    /// </para>
+    /// </summary>    
+    public class DefaultLambdaJsonSerializer : ILambdaSerializer
+    {
+        private const string DEBUG_ENVIRONMENT_VARIABLE_NAME = "LAMBDA_NET_SERIALIZER_DEBUG";
+        private readonly JsonSerializerOptions _options;
+        private readonly bool _debug;
+
+        /// <summary>
+        /// Constructs instance of serializer.
+        /// </summary>        
+        public DefaultLambdaJsonSerializer()
+        {
+            _options = new JsonSerializerOptions()
+            {
+                IgnoreNullValues = true,
+                PropertyNameCaseInsensitive = true,
+                PropertyNamingPolicy = new AwsNamingPolicy()
+            };
```

Comment:
A bit more modern notation, but functionality the same:
```
            _options = new JsonSerializerOptions()
            {
                IgnoreNullValues = true,
                PropertyNameCaseInsensitive = true,
                PropertyNamingPolicy = new AwsNamingPolicy(),
                Converters = 
                {
                    new DateTimeConverter(),
                    new MemoryStreamConverter(),
                    new ConstantClassConverter()
                }
            };
```

Reply:
Done